R/as_workflow_set.R

```diff
@@ -25,7 +75,10 @@ as_workflow_set <- function(...) {
          info = purrr::map(workflow, get_info),
          option    = purrr::map(1:nrow(res), ~ new_workflow_set_options())
       )
-   res$result <- unname(object)
+   res$result <- vector(mode = "list", length = nrow(res))
+   if (length(tune_index) > 1) {
+      res$result[tune_index] <- unname(object[tune_index])
+   }
```

Comment:
Shouldn't it be `>= 1?` Nevertheless, I would just do this unconditionally. i.e. remove the `if()` condition.